app/app/urls.py

```diff
@@ -109,6 +109,7 @@
     # dashboard views
     re_path(r'^onboard/(?P<flow>\w+)/$', dashboard.views.onboard, name='onboard'),
     re_path(r'^onboard/contributor/avatar/?$', dashboard.views.onboard_avatar, name='onboard_avatar'),
+    url(r'^postComment/', dashboard.views.postComment, name='postComment'),
```

Comment:
What do you think about changing the naming here to just `comment` since the post is implied as we're creating a comment resource?

Reply:
sure, can do.